docs/components/movement-sensor/_index.md

```diff
@@ -533,8 +546,23 @@ accuracy = await my_movement_sensor.get_accuracy()
 
 **Returns:**
 
-- [(map\[string\]float32)](https://go.dev/blog/maps): The accuracy and/or precision of the sensor, if supported.
-  Contents depend on sensor model.
+- [(Accuracy)](https://pkg.go.dev/go.viam.com/rdk/components/movementsensor#Accuracy): The precision and reliability metrics of the movement sensor, which vary depending on model.
```

Comment:
```suggestion
- [(Accuracy)](https://pkg.go.dev/go.viam.com/rdk/components/movementsensor#Accuracy): The reliability metrics of the movement sensor, which vary depending on model.
```

Reply:
@susmitaSanyal Can you confirm if wording here and below is more accurate?

Reply:
It should stay as it is. Accuracy contains data that indicates both precision and reliability.

Reply:
I was thinking that reliability includes precision so it'd be redundant to say both but I guess that's not right